### PR TITLE
2018 Q4 OKR Planning

### PR DESCRIPTION
**Pull request by diasdavid**
Ref: https://github.com/ipfs/pm/issues/698

- [2018 Q3 Retrospective Document](https://docs.google.com/document/d/1ITVW5yysj0bZiluEKasQ57edpNwOZ-P5GQGgoclwQno/edit#heading=h.906pmio05dzr). Please complete this ASAP.
- [2018 OKR Spreadsheet](https://docs.google.com/spreadsheets/d/1Lfd91hi3nFlLRS1r-FHvK2ip2Ll6ukraufCgepw43bw/edit#gid=754421585)

@vmx et al, you know the drill :)

### CODE REVIEW

File: OKR.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q4
```
[the remark this answers is not in the document]
Comment (diasdavid) [reply OKR.md]: @vmx that's the intention, actually. Check the flow that happens on js-ipfs land.

[the remark this answers is not in the document]
Comment (vmx) [reply OKR.md]: In js-ipld land there is not really a flow as it's a one man show. I just put my OKRs into the spreadsheet and that's it.

But it's OK for me to update this file accordingly in two weeks.

[the remark this answers is not in the document]
Comment (diasdavid) [reply OKR.md]: > In js-ipld land there is not really a flow as it's a one man show.

Not the case :) You have me, @Stebalien, @davidad, @mikeal (who has been spending time doing IPLD things), @zcstarr (who has contributed in the past and can participate in the discussion) and then you have users such as @b5, @pgte, @satazor and others that can give you valuable feedback on the plan for the quarter.

[the remark this answers is not in the document]
Comment (zcstarr) [reply OKR.md]: Happy to participate in the discussion, been a little away from the project for a short spell, got wrapped up in finishing up some work stuff, but looking forward to having time again to jump back in again soon and help out.

**Comment by vmx** [comment]
I've updated the spreadsheet linked above with the things I have in mind for Q4. If you'ev any comments or requests, feel free to post them here or just add items there and assign them to yourself.

**Comment by diasdavid** [comment]
@mikeal @Stebalien @momack2 can you review @vmx OKRs? 

@vmx I guess you didn't get a chance to present them yet, correct? //cc @steverichmond

**Comment by vmx** [comment]
@diasdavid we (@jbenet, @momack2, @Stebalien, @mikeal) reviewed the OKRs yesterday night. They should be all set for the overall OKR meeting later tonight.

**Pull request merged**